app/views/content_items/consultation.html.erb

```diff
@@ -0,0 +1,232 @@
+<%= content_for :page_class, @content_item.format.dasherize %>
+<%= content_for :title, @content_item.title %>
+
+<div class="grid-row">
+  <div class="column-two-thirds">
+    <%= render 'govuk_component/title',
+        context: t("content_item.format.#{@content_item.document_type}", count: 1),
+        title: @content_item.title,
+        average_title_length: 'long' %>
+  </div>
+</div>
+
+<%= render 'shared/withdrawal_notice', content_item: @content_item %>
+
+<div class="grid-row">
+  <div class="column-two-thirds">
+    <%= render 'govuk_component/metadata',
+        from: @content_item.from,
+        part_of: @content_item.part_of,
+        first_published: @content_item.published,
+        last_updated: @content_item.updated,
+        other: {
+          'Applies to' => @content_item.applies_to
+        },
+        see_updates_link: true
+    %>
+  </div>
+</div>
+<%= render 'shared/history_notice', content_item: @content_item %>
+
+<% if @content_item.unopened? %>
+  <div class="consultation-notice">
+    <h2>
+      This consultation opens <% if @content_item.opening_date_midnight? %>on<% else %>at<% end %> <time datetime="<%= @content_item.opening_date_time %>"><%= @content_item.opening_date %></time>
+    </h2>
+  </div>
+<% elsif @content_item.pending_final_outcome? %>
+  <div class="consultation-notice">
+    <h2>
+      We are analysing your feedback
+    </h2>
+    <p>Visit this page again soon to download the outcome to this public&nbsp;feedback.</p>
```

Comment:
There is subtle presentational behaviour here that is mixed in with the content. How about the following?

```html
<style>
  .nowrap { white-space: nowrap; }
</style>

<p>Visit this page again soon to download the outcome to this <span class="nowrap">public feedback.</span</p>
```